tools/stanford/flowsheet_cleaner.py

```diff
@@ -69,9 +84,12 @@ def convert_row(row: Mapping[str, str]) -> Optional[Dict[str, str]]:
         return None
 
 
-def get_new_sources(root: str, child: str) -> Set[str]:
+def get_new_sources(
```

Comment:
Can we call this something more clear than `get_new_sources` like `map_unknown_concepts`?